File: VVVVID-Downloader.ConsoleApp/VVVVID-Downloader.ConsoleApp/Episode.cs
namespace VVVVID_Downloader.ConsoleApp
{
    public class Episode
    {
        public int id { get; set; }
        public int season_id { get; set; }
        public int video_id { get; set; }
        public string number { get; set; }
        public string title { get; set; }
        public string thumbnail { get; set; }
        public string description { get; set; }
        public bool expired { get; set; }
        public bool seen { get; set; }
        public bool playable { get; set; }
        public int ondemand_type { get; set; }
        public int vod_mode { get; set; }
        public string videoLink { get; set; }
    }
}

File: angular-client/src/app/models/episode.ts
export class Episode {
    id: number;
    season_id: number;
    video_id: number;
    number: string;
    title: string;
    thumbnail: string;
    views: number;
    length: number;
    description: string;
    expired: boolean;
    seen: boolean;
    playable: boolean;
    ondemand_type: number;
    vod_mode: number;
    videoLink: string;
    embed_info: string;
    embed_info_sd: string;
    video_shares: number;
    video_likes: number;

    constructor(data: Partial<Episode>) {
        Object.assign(this, data);
    }
}
